express/features/mobile-verification-resend.feature

```diff
@@ -0,0 +1,19 @@
+Feature: A contact form to resend the verification code to mobile
+  Background:
+    Given that the user is on the `Check your phone` page
+
+  Scenario: User doesn’t receive their code
+    When they select the "Not received a text message?" link 
+    Then they should be directed to the following URL: "http://localhost:3000/create/resend-phone-code"
```

Comment:
These should use the `Then they should be directed to the following page: "/relative/path"` step from the shared-steps.  These will not work for test environments when we have them.  The URL form is for testing external websites rather than pages in our own application.  It's correct for Slack and the support form but not our own pages.